Files/Views/SettingsPages/Experimental.xaml

```diff
@@ -40,6 +40,12 @@
                     Header="Show item owner in properties"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding ShowFileOwner, Mode=TwoWay}" />
+
+                <ToggleSwitch
+                    x:Uid="SettingsLoadShellContextMenuItems"
+                    Header="Show explorer's context menu items in Files"
```

Comment:
```suggestion
                    Header="Show shortcut menu handlers from Windows Explorer"
```

Reply:
Thought it would be better to have some clarity about which items we support displaying alongside standard operations.